adoc/chapters/programming_interface.adoc

```diff
@@ -6528,18 +6528,20 @@ parameter: [code]#access_mode::read#, [code]#access_mode::write# and
 | Access target | Meaning
 | [code]#target::device#
     | Access a buffer from a <<sycl-kernel-function>> via device global memory.
+      Also used to get a native backend handle to the buffer from within a host
+      task, to be used on device.
 | [code]#target::host_task#
-    | Access a buffer from a <<host-task>>.
+    | Access a buffer's data on host from within a <<host-task>>.
 |====
 
-Programs which specify the access target as [code]#target::device# and then
-capture the [code]#accessor# in a <<host-task>> can only use the accessor for
-interoperability through the [code]#interop_handle#, any other uses result in
-undefined behavior.
-
-Programs which specify the access target as [code]#target::host_task# and then
-use the [code]#accessor# from a <<sycl-kernel-function>> result in undefined
-behavior.
+When an [code]#accessor# is used from within a host task, the use of the
```

Comment:
Thanks have added back in.

Reply:
Thanks.  I think it would help to but a paragraph after that first sentence, though.  The first sentence describes limitations when using `accessor` in a kernel function, while the next three sentences describe limitations when using `accessor` in a host task.

Reply:
Thanks have changed